cql2pgjson/src/main/java/org/folio/cql2pgjson/CQL2PgJSON.java

```diff
@@ -560,6 +580,55 @@ private String pg(CQLTermNode node) throws QueryValidationException {
     return index2sql(node.getIndex(), node);
   }
 
+  private String pgSubQuery(CQLTermNode node, List<DbFkInfo> fks, boolean childToParent) throws QueryValidationException {
```

Comment:
Can you rename fks to a self-explanatory name, for example foreignKeys?
Can you add a javadoc comment about childToParent including the meaning of false?

Reply:
I'll have no object to short local names